cmd/lora-gateway-bridge/cmd/configfile.go

```diff
@@ -191,6 +191,24 @@ marshaler="{{ .Backend.MQTT.Marshaler }}"
     # public-key.pem with this device / gateway in Google Cloud IoT Core.
     jwt_key_file="{{ .Backend.MQTT.Auth.GCPCloudIoTCore.JWTKeyFile }}"
 
+    # Azure IoT Hub
```

Comment:
Similarly to GCP, I believe this requires the template interpolation.

E.g, `device_id="{{ .Backend.MQTT.Auth.AzureIoTHub.DeviceID }}"`

Reply:
Yep, that's correct.